ci/images/debian10/Dockerfile

```diff
@@ -42,7 +42,6 @@ ENV PATH=/usr/local/go/bin:$GOPATH/bin:$PATH
 
 # Install Go dependencies
 ENV GO111MODULE=on
-RUN go get mvdan.cc/sh/v3/cmd/shfmt
 RUN go get github.com/goreleaser/nfpm/cmd/nfpm@v1.9.0
```

Comment:
If we're able to drop `nfpm` next, does this mean we can drop the dependency on go?

Reply:
I hope so! 🤞 tbh, I don't know what `nfpm` is used for, but hopefully we can drop that and Go soon! I'll add a note to the Clean Up issue to investigate

Reply:
seems to be something release-related, I think we should clean up all of those images, particularly in light of the fact that CentOS changed their model recently with CentOS Stream

Reply:
Good call. I'll add a note for that as well.